nessie/multipart_test.go

```diff
@@ -1,28 +1,25 @@
-// +build systemtests
```

Comment:
Remember to return these..

Reply:
wanted the IDE and go test to check the code so I removed the build tag and enabled running these tests with a flag --system-tests.
think that it will enable better development experience than using a specific build tag